src/NHapi.Base/Parser/PipeParser.cs

```diff
@@ -69,11 +79,25 @@ public PipeParser()
         public PipeParser(IModelClassFactory factory)
             : base(factory)
         {
+            ParserConfiguration = new ParserConfiguration();
+        }
+
+        /// <summary>
+        /// Creates a new PipeParser.
+        /// <param name="parserConfiguration">Contains configuration that will be applied when parsing.</param>
+        /// </summary>
+        public PipeParser(IModelClassFactory factory, ParserConfiguration parserConfiguration)
+            : base(factory)
+        {
+            ParserConfiguration = parserConfiguration;
         }
 
         /// <inheritdoc />
         public override string DefaultEncoding => "VB";
 
+        public ParserConfiguration ParserConfiguration { get; }
```

Comment:
I realise you did this because this is how hapi does it but can this be made private? not sure it needs to be public.